File: PGS/Lv.0/겹치는 선분의 길이.cpp
#include <string>
#include <vector>

using namespace std;

int a[201] = { 0, };

int solution(vector<vector<int>> lines) {
	int answer = 0;

	for (auto line : lines) {
		for (int i = line[0]; i < line[1]; i++) {
			a[i + 100]++;
		}
	}

	for (int i = 0; i <= 201; i++) {
		if (a[i] >= 2) {
			answer++;
		}
	}

	return answer;
}

File: PGS/Lv.0/JAVA/겹치는 선분의 길이.java
class Solution {
    public int solution(int[][] lines) {

        int[] count = new int[201];
        for(int[] line : lines) {
            for(int i = line[0]; i < line[1]; i++) {
                count[i + 100]++;
            }
        }
        
        int answer = 0;
        for(int c : count) {
            if(c >= 2) answer++;
        }
        
        return answer;
    }
}